File: pos-soap-node/consumerCalculadora.js
var soap = require('soap');
var url = 'http://localhost:8080/Calc/CalculadoraSOAP?WSDL';
var dados = {'c':2, 'd':4};
soap.WSDL.prototype.ignoredNamespaces = [];
soap.createClient(url, function(err, client) {
    client.somar(dados,function(err,result,res){
         console.log('Método somar:',result.resposta);
        //  console.log('Retorno:',result);
        //  console.log('Resposta:',res);
    })
});

File: pos-soap-python/consumerCalculadora.py
from suds.client import Client
from suds.cache import NoCache
url = "http://localhost:8080/Calc/CalculadoraSOAP?WSDL"
client = Client(url, cache=NoCache())
#invocando os metodos
print client.service.somar(3,2);
# print client.service.SomarNumeros(3,2);
#exibindo os servicos
# print client
